### ISSUE TITLE
text file location after creation

### ISSUE THREAD

**Issue by trippinshrumes** [rclone/rclone#2429]
#### What is the problem you are having with rclone?

2 systems, same setup.

rclone settings are

```
[gdrive]
type = drive
client_id = XXX
client_secret = XXX
scope = drive
root_folder_id =
service_account_file =
token = {"access_token":"*****","token_type":"Bearer","refresh_token":"****","expiry":"2018-07-31T22:18:29.59101748-05:00"}

[cache]
type = cache
remote = gdrive:Cloud
plex_url =
plex_username =
plex_password =
chunk_size = 10M
info_age = 48h
chunk_total_size = 10G
```

My upstart service file is

```
[Unit]
Description=RClone Service
AssertPathIsDirectory=/DATA/FUSE/Rclone
After=network-online.target
Wants=network-online.target
[Service]
Type=simple
ExecStart=/usr/bin/rclone mount cache: /DATA/FUSE/Rclone \
--allow-other \
--dir-cache-time=160h \
--cache-chunk-size=10M \
--cache-info-age=168h \
--cache-workers=5 \
--buffer-size=500M \
--attr-timeout=1s \
--syslog \
--umask 002 \
--rc \
--cache-tmp-upload-path /DATA/FUSE/cache_rclone_upload \
--cache-tmp-wait-time 30m \
--log-level INFO
ExecStop=/usr/bin/sudo /bin/fusermount -uz /DATA/FUSE/Rclone
Restart=on-abort
User=plex
Group=plex
```

--------------

When mounted, I change in to Rclone directory and I am presented with my Movie, Tv folders etc... If I then go in to my Movies or Tv folder and I create a text file, (nano test) and fill this file with basic letters, save and exit. going back one folder level will reveal the text file there. This does not happen with folders.

This is replicable on both of my systems, both running ubuntu 18.04

#### What is your rclone version (eg output from `rclone -V`)

1.42
#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
ubuntu 18.04 64

####  Which cloud storage system are you using? (eg Google Drive)

Google Suite
#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

it's up above in my service file
#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

Unfortunately I am unable to produce a log file at this time, my apologies. Although this is easily replicable.

**Comment by ncw** [comment]
I tried to replicate this with a mount on `/mnt/tmp` like this, but couldn't - what are you doing differently?

```
$ ls /mnt/tmp/
hello1.txt  hello2.txt
$ mkdir /mnt/tmp/potato
$ cd /mnt/tmp/potato
$ nano test-file
$ ls -l
total 1
-rw-rw-r-- 1 ncw ncw 21 Aug  1 15:06 test-file
$ cd ..
ncw@dogger:/mnt/tmp$ ls -l
total 1
-rw-rw-r-- 1 ncw ncw 6 May 31 13:48 hello1.txt
-rw-rw-r-- 1 ncw ncw 6 May 31 13:49 hello2.txt
drwxrwxr-x 1 ncw ncw 0 Aug  1 15:05 potato
ncw@dogger:/mnt/tmp$ 
```

**Comment by trippinshrumes** [comment]
you would need to 

`rclone mount cache: /mnt/tmp`

Inside of your /mnt/tmp it would ideally show your tvshows/movies folders etc, Going in to the TV shows or movies folder and then writing a text file will result in the text file still being editable from movies or tv shows

For me I have the mount to my rclone folder, this is my other system (different than the one referenced so different mountpoint, everything else is the same)


`plex@ZZeus:~/library/Rclone$ ls
 acd-check.txt  'converted movies'   Movies  'Music Flac'     rmgdrive.rtxt  'Stand Ups'   x4    x55
 Books           index.tsv           Music   'Music Videos'   Sports         'Tv Shows'    x44
plex@ZZeus:~/library/Rclone$ cd Books
plex@ZZeus:~/library/Rclone/Books$ ls
'Anne Rice'       'Douglas Preston'  'Lincoln Child'  'Sarah Ash'
'Dean R. Koontz'  'Jim Butcher'      'Louis LAmour'   'Stephen King'
plex@ZZeus:~/library/Rclone/Books$ nano testfile
plex@ZZeus:~/library/Rclone/Books$
plex@ZZeus:~/library/Rclone/Books$ ls
'Anne Rice'       'Douglas Preston'  'Lincoln Child'  'Sarah Ash'
'Dean R. Koontz'  'Jim Butcher'      'Louis LAmour'   'Stephen King'
plex@ZZeus:~/library/Rclone/Books$ cd ..
plex@ZZeus:~/library/Rclone$ ls
 acd-check.txt       index.tsv  'Music Flac'     Sports      'Tv Shows'   x5    x8         xxx.txt
 Books               Movies     'Music Videos'  'Stand Ups'   x4          x55   xccc
'converted movies'   Music       rmgdrive.rtxt   testfile     x44         x7    xxx2.txt`

**Comment by trippinshrumes** [comment]
Also if it matters any, I only noticed this because sonarr was unable to verify write access to the Tv Shows folder, but was able to verify write access to the root mount folder, which in my case is /DATA/FUSE/Rclone on the one, and /home/plex/library/Rclone on the other.

**Comment by ncw** [comment]
Thanks, great explanation.  It is pretty much exactly what I did so I'm surprised at the different results...

> would getting a log file help with seeing what's happening here?

Yes that would help.  If you can create a log with `-vv --logfile rclone.log` then we can check exactly what nano did.  The log might be quite large so can you zip it and attach it here?

**Comment by trippinshrumes** [comment]
I'm having issues generating the log file. It writes to syslog fine, although it would be more beneficial to have it create it's own log for this. 

using rclone.log I get a permission error. I even went as far as giving the user write permission to the entire /var/log folder, this has been tested and verified. 

I'm not sure where this is trying to store the log. The documentation does not specify default log path other than --syslog. 

I have tried specifying location with --logfile=$HOME/rclone.log  // --logfile=/home/plex/rclone.log // --logfile= /home/plex/rclone.log  as well as all three of these without the equal sign (--logfile) 

These have forced rclone to quit and error. I have also tried removing --syslog while inputting these. 

did the syntax change from what I'm seeing?
